src/gtl/components/tile-view/tile-view.html

```diff
@@ -1,25 +1,25 @@
 <div class="miq-tile-section">
-  <div ng-if="tileCtrl.settings.isLoading" class="spinner spinner-lg"></div>
-  <div class="row miq-pagination" ng-if="tileCtrl.rows && tileCtrl.rows.length !== 0">
-    <div ng-if="tileCtrl.rows.length !== 0" class="miq-select-all col-md-3 col-ld-3">
-      <label>{{tileCtrl.settings.selectAllTitle}}: </label>
-      <input type="checkbox" ng-model="isChecked" ng-click="tileCtrl.onCheckAllTiles(isChecked)" title="{{tileCtrl.settings.selectAllTitle}}" />
+    <div ng-if="tileCtrl.settings.isLoading" class="spinner spinner-lg"></div>
```

Comment:
Should ARIA attributes be added for accessibility?
Probably a separate PR if true.

^^ @martinpovolny @himdel